hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/ReconTaskSchemaDefinition.java

```diff
@@ -64,6 +64,7 @@ private void createReconTaskStatusTable(Connection conn) {
         .column("task_name", SQLDataType.VARCHAR(766).nullable(false))
         .column("last_updated_timestamp", SQLDataType.BIGINT)
         .column("last_updated_seq_number", SQLDataType.BIGINT)
+        .column("last_task_successful", SQLDataType.BIT)
```

Comment:
Also use as INTEGER datatype, it will be easy to integrate with existing hadoop metrics.

Reply:
Addressed in [5248373](https://github.com/apache/ozone/pull/7517/commits/5248373fede0a47e1475ef14730e4b443445b55d)